### ISSUE TITLE
exclude whole folders in dartdoc_options.yaml

### ISSUE THREAD

**Issue by Freundschaft** [dart-lang/dartdoc#2236]
I am developing a flutter application, and its very annoying that all my screens are included in my dartdocs, where I only want my components to be documented.
Is there any way I can exclude a complete folder from the dartdoc generation? currently I need to exclude every single library by hand, which results in a huge list of files i need to explicitly exclude.

I am looking for something like this:

```
dartdoc:
  exclude: ['package:myapp/screens/**/**']
```

**Comment by jcollins-g** [comment]
Currently the syntax doesn't support that, but it really should.

**Comment by Freundschaft** [comment]
yes, I see an issue especially with the way dart is used in flutter, where usually for every screen, a seperate .dart file is created, which dart treats as a seperate mini-library as far as I understand.
this results in a lot of unnecessary documentation
I tried adding the "@nodoc" attribute to screen classes, but currently dartdoc doesnt skip generation for a library, even if every member of that library is labeled "@nodoc", which results in a documentation generated for that library with no content, which is not what I need.

**Comment by jcollins-g** [comment]
If you place the screens directory under 'lib/src', the libraries will not be documented and considered private -- this is a general convention for dart packages that dartdoc understands.

Also, `@nodoc` should work if it is dropped into the library level documentation comment and obviate the need for tagging all your classes... if not that's a bug.

**Comment by Freundschaft** [comment]
perfect, thanks I will try that.

**Comment by mgesmundo** [comment]
Some news about this feature?

**Comment by srawlins** [comment]
Sorry, no news.
